clients/fides-js/docs/interfaces/Fides.md

```diff
@@ -0,0 +1,237 @@
+- value: `true` or `false`, depending on whether or not the current user
+has consented to the notice
+
+Note that FidesJS will automatically set default consent preferences based
+on the type of notice - so, for example a typical "opt-in" analytics notice
+will be given a default value of `false`. This allows very writing very
+simple (and readable!) code to check a user's consent preferences.
+
+The specific keys provided in the `Fides.consent` property are determined
+based on your Fides configuration, and are provided to the browser based on
+the user's location, property ID, etc.
+
+**`Example`**
+
+A `Fides.consent` value showing the user has opted-out of data sales & sharing:
+```ts
+{
+  "data_sales_and_sharing": false
+}
+```
+
+**`Example`**
+
+A `Fides.consent` value showing the user has opted-in to analytics, but not marketing:
+```ts
+{
+  "analytics": true,
+  "marketing": false
+}
+```
+
+___
+
+### fides\_string
+
+• `Optional` **fides\_string**: `string`
+
+User's current consent string(s) combined into a single value. Currently,
+this is used by FidesJS to store IAB consent strings from various
+frameworks such as TCF, GPP, and Google's "Additional Consent" string.
+
+**`Example`**
+
+Example `fides_string` showing a combination of:
+- IAB TC string: `CPzHq4APzHq4AAMABBENAUEAALAAAEOAAAAAAEAEACACAAAA`
+- Google AC string: `1~61.70`
+```ts
+console.log(Fides.fides_string); // CPzHq4APzHq4AAMABBENAUEAALAAAEOAAAAAAEAEACACAAAA,1~61.70
+```
+
+___
+
+### initialized
+
+• **initialized**: `boolean`
+
+Whether or not FidesJS has finished initialization and has loaded the
+current user's experience, consent preferences, etc.
+
+NOTE: To be notified when initialization has completed, you can subscribe
+to the `FidesInitialized` event. See [FidesEvent](FidesEvent.md) for details.
+
+___
+
+### showModal
+
+• **showModal**: () => `void`
+
+Display the FidesJS modal component on the page, if the current user's
+session (location, property ID, etc.) matches an `experience` with a modal
+component. If the `experience` does not match, this function has no effect
+and can be called safely at any time.
+
+This function is designed to be used to programmatically show the FidesJS
+modal via an `onclick` handler on a "modal link" element on the page.
+However, since the modal is optional, this link should only be shown when
+applicable. To make it easy to dynamically show/hide this "modal link",
+FidesJS will automatically add the CSS class `fides-overlay-modal-link-shown`
+to the `<body>` when applicable. This class can then be used to show/hide a
+link on the page via CSS rules - see the example below!
+
+When not used as a click handler, `Fides.showModal()` can be called
+programmatically at any time from your own custom Javascript logic as
+desired.
+
+**`Example`**
+
+Showing the FidesJS modal via an `onclick` handler on a custom link element:
+```html
+<a href="#" class="my-custom-link" onclick="Fides.showModal()">
```

Comment:
Yes, that's a solid suggestion. FWIW the use of `<a>` is mostly because most sites include a set of footer links to their privacy policy, TOS, etc., and that tends to be the "right" place to put this. But I can show a couple options here.